bgpd/rfapi/rfapi_import.c

```diff
@@ -3530,7 +3530,9 @@ void rfapiBgpInfoFilteredImportVPN(
 			 * Compare types. Doing so prevents a RFP-originated
 			 * route from matching an imported route, for example.
 			 */
-			assert(bi->type == type);
+			if (bi->type != type) /* should be handled by RDs, but warn for now */
```

Comment:
If I receive 10k routes like this would I get 10k zlog_warns?  We might want to put this behind a debug check?